handheld.css

```diff
@@ -0,0 +1,107 @@
+@media screen and (max-width: 400px) {
+  body {
```

Comment:
- [ ] As requested by the previous reviewer, the breakpoint in the desktop version should be at least between 768px to 1440px. Kindly fix it.
- [ ]  Also the requirements are clear regarding implementing a mobile-first approach to web development. Refer to the previous reviewer's comments. So instead of a max-width media query, we use the min-width media query. [check out this link](https://nikitahl.com/difference-between-min-width-vs-max-width)